src/internal/hooks/use-visual-mode/index.ts

```diff
@@ -21,6 +26,9 @@ function detectVisualRefresh() {
 
 export function useVisualRefreshDynamic() {
   if (visualRefreshState === undefined) {
+    if (typeof window !== 'undefined' && window[Symbol.for('isVisualRefresh')]?.()) {
```

Comment:
Symbol names are global for all objects on the page, so we need to have a scoped name. Following the convention in [our runtime API](https://github.com/cloudscape-design/components/blob/df04533aa56157985792e1ceb8aaf965a77a193d/src/internal/plugins/api.ts#L6C20-L6C48), I would recommend `Symbol.for('awsui-visual-refresh-flag')`.

This is a moment where the longer name, the better